action.yml

```diff
@@ -124,8 +124,12 @@ inputs:
     description: "Enable checkout action to (re-)download your repository prior to running the pipeline"
     default: true
 
+  enable-checkout-submodules:
+    description: "Enable submodules for the checkout action (false|true|recursive)"
+    default: recursive
```

Comment:
Should we use `recursive` or `true` as default? `recursive` will change the default behavior of docker-ros.

Reply:
Recursive has been the default for GitLab. Either way, I would make it consistent.